localstack-core/localstack/packages/terraform.py

```diff
@@ -7,18 +7,18 @@
 from localstack.utils.files import chmod_r
 from localstack.utils.platform import get_arch
 
-TERRAFORM_VERSION = "1.1.3"
+TERRAFORM_VERSION = os.getenv("LS_TERRAFORM_VERSION", "1.5.7")
```

Comment:
We don't use prefixes for our environment variables anywhere else, so I think here it should also not be prefixed.
```suggestion
TERRAFORM_VERSION = os.getenv("TERRAFORM_VERSION", "1.5.7")
```

Reply:
I prefixed it on the _assumption_ that terraform might actually pay attention to an environment variable like `TERRAFORM_VERSION`, but upon actually [checking it](https://developer.hashicorp.com/terraform/cli/config/environment-variables), it absolutely does not.

tl;dr good catch, great call